seatunnel-e2e/seatunnel-connector-v2-e2e/connector-jdbc-e2e/connector-jdbc-e2e-part-3/src/test/java/org/apache/seatunnel/connectors/seatunnel/jdbc/JdbcHiveIT.java

```diff
@@ -43,7 +44,7 @@ public class JdbcHiveIT extends AbstractJdbcIT {
 
     private static final String HIVE_DATABASE = "default";
 
-    private static final String HIVE_SOURCE = "e2e_table_source";
+    private static final String HIVE_SOURCE = "hive_e2e_source_table";
```

Comment:
why change this?

Reply:
In this case, the create statement is `hive_e2e_source_table`